Frontend/ui-customer/src/Resources/Animation/Config.purs

```diff
@@ -95,3 +95,6 @@ translateFullYAnimWithDurationConfig duration = animConfig {
 , toY = 0
 , interpolator = PrestoAnim.EaseInOut
 }
+
```

Comment:
@dhruvSHA256 move this function to constants.purs, here we only have configs related to animation. And also this value is different for iOS , it is 19.0 for iOS and 17.0 for android.
Test this once in both android and iOS.